docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md

```diff
@@ -315,62 +334,95 @@ The enterprise data model is now ready to be validated in `neat`. David will val
 
-<!-- TODO:
-- Explain a bit more about disconnect in classes from wind expert
-- Why we switch from string to actual object to represent location
-- Why we used CIM, GeoJSON, and WGS84 standards to define the enterprise data model
-- ... -->
 
-### Iterating over the Sheet
 
-Looking over the `Properties` sheet, David notice that `WindGenerator` from Emma and `WindTurbine` from Jon are
-very similar. He decides to prompt Emma and Jon for clarification. After a short discussion, they decide that
-`WindGenerator` and `WindTurbine` are the same thing, and they decide to use `WindTurbine` as the class name.
+### Visualize Semantic Data Model
+Due to additional information that David as an information architect has added to the enterprise data model,
+David can exported Rules as semantic data model which consists of:
 
-An alternative approach would be that keep both classes, but add a property to the `WindGenerator` class that
-specifies that it is a type of `WindTurbine`. This would be done by using the `Parent Class` column in the
-`classes` sheet.
+- Ontology presented as a set of RDF triples using [OWL](https://www.w3.org/OWL/) standard
+- Shape object constraints presented as a set of RDF triples using [SHACl](https://www.w3.org/TR/shacl/) standard
 
-| Class           | Description                                         | Parent Class   |
-|-----------------|-----------------------------------------------------|----------------|
-| WindTurbine     | A device that converts wind to electrical energy    | WindGenerator  |
+The derived data model can be exported as so-called Turtle file, which is a standard way to represent RDF data.
+This file can be used in any ontology tool, such as [Protégé](https://protege.stanford.edu/), while combination of
+both OWL and SHACL can be used to in a tool such as [SHACL Play!](https://shacl-play.sparna.fr/play/).
 
+For purpose of this demonstration David will actually use `neat` to load generated semantic data model into
+internal RDF store and visualize it using `neat` UI. To do this, David selects `Visualize Semantic Data Model` workflow
+which unlike the previous is a bit more demanding as it requires configuration of additional steps, specifically:
 
-### Extending the Sheet
+- `RulesToSemanticDataModel` : which will convert `Rules` into semantic data model and store it as a Turtle file
+- `GraphStoreConfiguration`: which will configure the internal RDF store
+- `GraphFromRdfFile` : which will load the semantic data model (the Turtle file) into the internal RDF store
 
-David will also add two new columns to the `Properties` sheet. The first column is called `Source`, and it is used
-to specify where the statement comes from, or what standard that matches the statement. The second column is called
-`MatchType`, and tells whether the source is partially or fully matching the statement.
+The process of configuration and execution of these steps is shown in the video below:
+https://youtu.be/_iTQatLBpns
```

Comment:
Should this also be put into an `iframe`?

Reply:
it should not be there at all :D